src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1137,16 +1137,15 @@ def astype(
         new_query_compiler = self._query_compiler.astype(col_dtypes, errors=errors)
         return self._create_or_update_from_compiler(new_query_compiler, not copy)
 
-    @base_not_implemented()
     @property
     def at(self, axis=None):  # noqa: PR01, RT01, D200
         """
         Get a single value for a row/column label pair.
         """
         # TODO: SNOW-1119855: Modin upgrade - modin.pandas.base.BasePandasDataset
-        from .indexing import _LocIndexer
+        from .indexing import _AtIndexer
```

Comment:
This does deviate from Modin, but I think this implementation with `_AtIndexer` and the error checking is much cleaner

Reply:
I agree